src/main/java/com/parasoft/findings/jenkins/coverage/api/metrics/steps/CoverageXmlStream.java

```diff
@@ -244,8 +247,8 @@ NavigableMap<K, V> unmarshal(final String value) {
                                 StringUtils.substringAfter(marshalledValue, ':'));
                         map.put(entry.getKey(), entry.getValue());
                     }
-                    catch (IllegalArgumentException exception) {
-                        // ignore
+                    catch (IllegalArgumentException exception) { // parasoft-suppress OWASP2021.A9.LGE "This is intentionally designed to ensure exceptions during coverage report processing don't cause the build to fail."
```

Comment:
The violation is no longer reported when caught exceptions is logged, so we can remove the suppression "OWASP2021.A9.LGE".